example/android/app/build.gradle

```diff
@@ -251,6 +251,9 @@ android {
 dependencies {
     implementation fileTree(dir: "libs", include: ["*.jar"])
     //noinspection GradleDynamicVersion
+
+    implementation "androidx.localbroadcastmanager:localbroadcastmanager:1.1.0"
```

Comment:
Please add a short comment on why this is needed. i.e. this is needed for the offline feature.

Reply:
@matamegger Where should this comment exist?  

This is in the required setup for the android SDK found here: https://developer.bitmovin.com/playback/docs/working-with-offline-playback-android, so I _believe_ that means we should add a Feature Guide under the react native section for the offline stuff.  I don't think I can do that though.

Reply:
Good catch. Indeed we should extend the getting started guide for react native 👍 

Please still add a code comment above the dependency. I think something like this explains the need for the particular dependency:
`// Only needed if the offline feature is used`

Reply:
@matamegger Sounds good.  I'll get the comment added.